<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>341</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>Calc pro max</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>321</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="2" colspan="3">
       <widget class="QTextBrowser" name="textBrowser_3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="placeholderText">
         <string extracomment="123"/>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QPushButton" name="pushButton_9">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="16" column="3">
       <widget class="QPushButton" name="pushButton_15">
        <property name="text">
         <string>,</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="14" column="2">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="14" column="4">
       <widget class="QPushButton" name="pushButton_19">
        <property name="text">
         <string>mod</string>
        </property>
       </widget>
      </item>
      <item row="12" column="3">
       <widget class="QPushButton" name="pushButton_12">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_33">
          <property name="text">
           <string>cos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_34">
          <property name="text">
           <string>sin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_28">
          <property name="text">
           <string>tan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_31">
          <property name="text">
           <string>ln</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_30">
          <property name="text">
           <string>log</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="13" column="3">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="14" column="3">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="3" colspan="3">
       <widget class="QTextBrowser" name="textBrowser_4">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QPushButton" name="pushButton_25">
        <property name="text">
         <string>e</string>
        </property>
       </widget>
      </item>
      <item row="12" column="4">
       <widget class="QPushButton" name="pushButton_11">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <widget class="QPushButton" name="pushButton_17">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="16" column="4">
       <widget class="QPushButton" name="pushButton_24">
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="13" column="4">
       <widget class="QPushButton" name="pushButton_20">
        <property name="text">
         <string>^</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QPushButton" name="pushButton_16">
        <property name="text">
         <string>ac</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_42">
          <property name="text">
           <string>acos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_40">
          <property name="text">
           <string>asin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_41">
          <property name="text">
           <string>atan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_38">
          <property name="text">
           <string>sqrt</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="15" column="4">
       <widget class="QPushButton" name="pushButton_23">
        <property name="text">
         <string>(</string>
        </property>
       </widget>
      </item>
      <item row="15" column="2">
       <widget class="QPushButton" name="pushButton_8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QPushButton" name="pushButton_13">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="4">
       <widget class="QTextBrowser" name="textBrowser_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>37</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 17pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="markdown">
         <string notr="true"/>
        </property>
        <property name="html">
         <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:17pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="overwriteMode">
         <bool>true</bool>
        </property>
        <property name="acceptRichText">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
        <property name="openLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QPushButton" name="pushButton_14">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="15" column="3">
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="11" column="4">
       <widget class="QPushButton" name="pushButton_18">
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="11" column="3">
       <widget class="QPushButton" name="pushButton_21">
        <property name="text">
         <string>del</string>
        </property>
       </widget>
      </item>
      <item row="16" column="2">
       <widget class="QPushButton" name="pushButton_10">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>x=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="markdown">
           <string/>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_26">
          <property name="text">
           <string>clear x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_22">
          <property name="text">
           <string>save to x</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_27">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>360</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>400</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>f(x)=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>450</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>graph mode</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>350</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhFormattedNumbersOnly</set>
    </property>
    <property name="maxLength">
     <number>260</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
